skype/skype-ps/skype/ConvertTo-JsonForPSWS.md

```diff
@@ -17,21 +17,98 @@ ConvertTo-JsonForPSWS [[-InputObject] <Object>] [-Compress] [-Depth <Object>] [-
 ```
 
 ## DESCRIPTION
-{{Fill in the Description}}
+The `ConvertTo-JsonForPSWS` cmdlet converts any object to a string in JavaScript Object Notation (JSON) format for PowerShell Web Services. The properties are converted to field names, the field values are converted to property values, and the methods are removed.
+
+You can then use the `ConvertTo-JsonForPSWS` cmdlet to convert a JSON-formatted string to a JSON object, which is easily managed in Windows PowerShell.
+
+Many web sites use JSON instead of XML to serialize data for communication between servers and web-based apps.
```

Comment:
Let's remove this sentence:

Many web sites use JSON instead of XML to serialize data for communication between servers and web-based apps.

Reply:
Done, thank you!